addons/mrp/views/stock_picking_views.xml

```diff
@@ -111,4 +111,15 @@
             </xpath>
         </field>
     </record>
+
+    <record id="view_picking_form_inherit_mrp" model="ir.ui.view">
+        <field name="name">stock.picking.form</field>
+        <field name="model">stock.picking</field>
+        <field name="inherit_id" ref="stock.view_picking_form"/>
+        <field name="arch" type="xml">
+            <field name="picking_type_id" position="attributes">
+                <attribute name="domain">[('code', '!=', 'mrp_operation')]</attribute>
+            </field>
```

Comment:
in the stock addon, stock_picking.py
``` python
    picking_type_id = fields.Many2one(
        'stock.picking.type', 'Operation Type', domain="[('code', '!=', 'mrp_operation')]",
```

it'll be easy for our multi company rebase